packages/katello/katello-installer/katello-installer-base.spec

```diff
@@ -117,3 +118,6 @@ ln -sf %{_datadir}/foreman-installer-katello/bin/katello-certs-check %{buildroot
 %doc README.*
 
 %changelog
+* Wed Jul 18 2018 Eric D. Helms <ericdhelms@gmail.com> 3.9.0-1.nightly
+Bump
```

Comment:
What's the difference between using `- ` and not?

Reply:
I think it's just a convention